Docker

ByteHamster opened the pull request:
Ich stell das mal rein, dass wir es auf dem Schirm haben

Dav1dK committed:
Add docker files

Dav1dK committed:
[docker] Copy dockerfiles to the root directory

Otherwise we can't COPY the files into the container.
Also fix the dockerfile name.

Dav1dK committed:
[docker] Don't rebuild containers everytime

Dav1dK committed:
[docker] Add docker instructions to README

Dav1dK committed:
[docker] Don't checkout monitorController branch

The branch is merged into master.

Dav1dK commented:
Ja mach das. Es braucht anscheinend eine aktuelle Version von Docker Compose. Macht das etwas?

Dav1dK commented:
Und bitte probiert es mal aus (und sagt mir unter welchen Versionen es läuft).

ByteHamster commented on docker-compose.yml:
```diff
@@ -0,0 +1,27 @@
+  simulation:
+    build:
+      context: .
+      dockerfile: simulationDockerfile
```
Ich hab keine Ahnung, wovon ich rede. Sollte es aber nicht für simulation und für monitoring die gleichen Dateien geben?

Dav1dK replied:
Nein das stimmt schon. Da beide etwas anderes starten, haben beide ein anderes Dockerfile. Wobei man vermutlich auch ein einziges Dockerfile verwenden könnte...
Ich versuch das in den nächsten Tagen mal.

ByteHamster replied:
Ah hab mich verlesen. Dachte das wäre beides mal simulation in dieser Datei

ByteHamster commented:
Mein nvidia grafiktreiber ist kaputt, aber immerhin hab ich Docker zum Laufen bekommen :P Dein PR funktioniert ;)

Dav1dK committed:
[docker] Adapt docker-compose.yml to the file version 1

Otherwise it doesn't work with the docker-compose version
present in ubuntu 16.04.
